sw/device/rom_exts/rom_ext_manifest.S

```diff
@@ -38,22 +39,20 @@ image_version:
 image_timestamp:
   .dword 0x0
 
-// Overridden by signing script
-image_signature_algorithm:
-  .word 0x0
-
 // Overriden by signing script
-image_signature_exponent:
+image_signature_public_key_exponent:
```

Comment:
This should rather read `image_signature_key_public_exponent`. I think what is meant is the _public exponent_ of the _signature key_ of the _image_.

Reply:
Yeah, sorry. 

The plan in the long-run is to auto-generate this file from manifest.hjson, which should make your life easier for this kind of change :)